src/components/test/common-tests.js

```diff
@@ -9,13 +11,23 @@ const createComponent = (Component, props = {}) => {
   return mount(<Component {...props}>This is child content</Component>);
 };
 
+function primaryElement(wrapper, elementSelector) {
+  if (elementSelector) {
+    return wrapper.find(elementSelector).getDOMNode();
+  }
+  return wrapper.childAt(0).getDOMNode();
+}
+
 /**
  * Set of tests common to all presentational components
  *
  * @typedef CommonTestOpts
  * @prop {string} [componentName] - Temporary affordance to prevent name
  *   collisions between updated components and legacy components with the same
  *   component function name.
+ * @prop {(component:FunctionComponent, props:object) => import('preact').VNode|ReactWrapper} [content] -
+ *   A function that returns the rendered output to test or an Enzyme wrapper
+ *   created using Enzyme's `mount` function.
```

Comment:
Various tests below assume that the result of `content` is always a wrapper, rather than a VNode. Also `content` sounds like an object rather than a function. Perhaps `createContent`?

Reply:
I thought the same thing! I copied the property name from the `client`'s support for an analogous feature for accessibility tests there. I like `createContent` better, too.